lua/ttt2/extensions/draw.lua

```diff
@@ -457,37 +457,118 @@ function draw.AdvancedText(text, font, x, y, color, xalign, yalign, shadow, scal
 	end
 end
 
+-- If there are no spaces, we have to cut the string at some point.
+-- To improve performance, we don't want to iterate over every single
+-- character. Therefore we assume the length based on an average first.
+local function InternalSplitLongWord(word, width, widthWord)
+	local charCount = fastutf8.len(word)
+	local wCharAverage = widthWord / charCount
+	-- decrease a bit to have tolerance & limit to at least 1, to prevent infinite loops
+	local charCountPerLine = math.max(1, math.floor(width / wCharAverage * 0.9))
+
+	local lines = { "" }
```

Comment:
```suggestion
	local lines = {}
```

Reply:
i think this is needed to always return a string

Reply:
in case the first char is a number? makes sense